src/gui/tray/Window.qml

```diff
@@ -470,6 +562,17 @@ Window {
                 height: Style.trayWindowHeaderHeight
                 spacing: 0
 
+                Accessible.role: Accessible.ListItem
+                Accessible.name: activityTextColumn.text
+                Accessible.description: qsTr("Activity item - if applicable opens local folder containing the corresponding sync object when clicked")

```

Comment:
Do we know which are clickable and which not? Cause then for ones where it would open the file locally we could just set the name to: "Open `activityTextColumn.text` locally" and no description is needed.